server/controllers/FileProfile.js

```diff
@@ -59,35 +59,9 @@ module.exports.getFileProfileOperation = async function getFileProfileOperation(
   oamLogService.recordOamRequest(req.url, req.body, responseCode, req.headers.authorization, req.method);
 };
 
-module.exports.getFileProfilePassword = async function getFileProfilePassword(req, res, next, uuid) {
-  let responseCode = responseCodeEnum.code.OK;
-  await FileProfile.getFileProfilePassword(req.url)
-    .then(function (response) {
-      responseBuilder.buildResponse(res, responseCode, response);
-    })
-    .catch(function (response) {
-      let sentResp = responseBuilder.buildResponse(res, undefined, response);
-      responseCode = sentResp.code;
-    });
-  oamLogService.recordOamRequest(req.url, req.body, responseCode, req.headers.authorization, req.method);
-};
-
-module.exports.getFileProfileUserName = async function getFileProfileUserName(req, res, next, uuid) {
-  let responseCode = responseCodeEnum.code.OK;
-  await FileProfile.getFileProfileUserName(req.url)
-    .then(function (response) {
-      responseBuilder.buildResponse(res, responseCode, response);
-    })
-    .catch(function (response) {
-      let sentResp = responseBuilder.buildResponse(res, undefined, response);
-      responseCode = sentResp.code;
-    });
-  oamLogService.recordOamRequest(req.url, req.body, responseCode, req.headers.authorization, req.method);
-};
-
-module.exports.putFileProfileFilePath = async function putFileProfileFilePath(req, res, next, body, uuid) {
+module.exports.putFileProfileFileName = async function putFileProfileFileName(req, res, next, body, uuid) {
```

Comment:
PUT filename is creating the following problem , 
```
{
  "code": 400,
  "message": "request.body['file-profile-1-0:file-name'] should match pattern \"^([A-Za-z0-9-_,\\s]{3,})\\.([A-Za-z]{3})$\"",
  "errors": [
    {
      "path": ".body['file-profile-1-0:file-name']",
      "message": "should match pattern \"^([A-Za-z0-9-_,\\s]{3,})\\.([A-Za-z]{3})$\"",
      "errorCode": "pattern.openapi.validation"
    }
  ]
}
```
for the input 
{
  "file-profile-1-0:file-name": "application-data.json"
}

Please have a look into the YAML file and kindly look for the possible rootcause.

Reply:
This commit does not using a updated yaml file so it is giving this error